udtube/modules.py

```diff
@@ -208,6 +197,8 @@ def __init__(
         **kwargs,
     ):
         super().__init__()
+        if not (use_upos or use_xpos or use_lemma or use_feats):
+            raise Error("No classification heads enabled")
```

Comment:
insanely minor but I have a preference toward
` if not any((use_pos, use_xpos, use_lemma, use_feats))`

(not needed right now, but we can probably do a check for any nn.Sequential objects in __dir__ but maybe YAGNI unless we are always adding more heads)

Reply:
i tried a few things and don't have a preference so i'll go do that now